include/rtcdcpp/PeerConnection.hpp

```diff
@@ -150,10 +150,13 @@ class PeerConnection {
 
   // DataChannel message parsing
   void HandleNewDataChannel(ChunkPtr chunk, uint16_t sid);
+  void HandleDataChannelClose(uint16_t sid);
   void HandleStringMessage(ChunkPtr chunk, uint16_t sid);
   void HandleBinaryMessage(ChunkPtr chunk, uint16_t sid);
 
   std::shared_ptr<Logger> logger = GetLogger("rtcdcpp.PeerConnection");
+ public:
+  void ResetSCTPStream(uint16_t stream_id);
```

Comment:
Fixed in https://github.com/chadnickbok/librtcdcpp/pull/24/commits/91db2d4e9fa1db09bc8a506c4e214561ff0ed65d. Let me know if this is okay.